lib/archethic/db/embedded_impl/chain_index.ex

```diff
@@ -40,7 +40,7 @@ defmodule Archethic.DB.EmbeddedImpl.ChainIndex do
     {:ok, %{db_path: db_path}}
   end
 
-  def code_change("1.0.7", state, _extra) do
+  def code_change("1.0.6", state, _extra) do
```

Comment:
It should be 1.0.7 I think

Reply:
no with 1.0.7 it will not work @Neylix experienced the same issue when doing manual upgrade in testnet

Reply:
I don't get it because the latest version deployed in the 1.0.7 and the chain index's cache off loading isn't deployed yet.

Reply:
I've seen with @Neylix, I think you can let it as before, it was just for the tests. :)

Reply:
ok done